src/mpid/ch4/shm/posix/eager/fbox/fbox_init.h

```diff
@@ -28,6 +28,17 @@
       description : >-
         The size of the array to store expected receives to speed up fastbox polling.
 
+    - name        : MPIR_CVAR_CH4_POSIX_EAGER_FBOX_MEMBIND
+      category    : CH4
+      type        : string
+      default     : "SYSTEM"
+      class       : device
+      verbosity   : MPI_T_VERBOSITY_USER_BASIC
+      scope       : MPI_T_SCOPE_ALL_EQ
+      description : >-
+        The memory binding type for fastboxes. Valid values are DDR and HBM for DDR
+        binding and HBM binding respectively.
+
```

Comment:
Instead of "string" type, you could use "enum" type. For example:
```
- name        : MPIR_CVAR_BCAST_INTRA_ALGORITHM
      category    : COLLECTIVE
      type        : enum
      group       : MPIR_CVAR_GROUP_COLL_ALGO
      default     : auto
      class       : device
      verbosity   : MPI_T_VERBOSITY_USER_BASIC
      scope       : MPI_T_SCOPE_ALL_EQ
      description : |-
        Variable to select bcast algorithm
        auto                                    - Internal algorithm selection
        binomial                                - Force Binomial Tree
        nb                                      - Force nonblocking algorithm
        scatter_recursive_doubling_allgather    - Force Scatter Recursive-Doubling Allgather
        scatter_ring_allgather                  - Force Scatter Ring
```

Reply:
Thanks this is much better